include/modem/modem_info.h

```diff
@@ -31,11 +31,20 @@ extern "C" {
 /** Size of the JSON string. */
 #define MODEM_INFO_JSON_STRING_SIZE 512
 
-/** RSRP offset value. */
-#define RSRP_OFFSET_VAL 140
+/** RSRP offset value when RSRP index is negative. */
```

Comment:
As discussed, these should not be used directly, so these can be removed and the values moved into the conversion macros.